packages/genie/package.py

```diff
@@ -67,9 +68,16 @@ def url_for_version(self, version):
 
     variant("lhapdf", default=True) 
 
+    # Use mainline spack gettext for lhapdf
     depends_on("lhapdf" , when="+lhapdf")
 
-    depends_on("root+pythia6")
+    # Issues caused by default root cxxstd being 11
+    # Separate issue caused by libxmp from +x requirement
```

Comment:
The `~x` variant on `root` should be applied externally via command line or environment requirements.